actions/kpops-runner/README.md

```diff
@@ -0,0 +1,27 @@
+# kpops-runner
+
+This action runs Kpops with the given config.
+
+## Input Parameters
+
+| Name              | Required | Default Value |  Type   | Description                                                                                                   |
+| ----------------- | :------: | :-----------: | :-----: | ------------------------------------------------------------------------------------------------------------- | --- |
+| mode              |    ✅    |       -       | string  | command mode used by Kpops (deploy, destroy, reset, clean)                                                    |
+| pipeline          |    ✅    |       -       | string  | Pipeline file to be run by Kpops publication                                                                  |     |
+| working-directory |    ❌    |       .       | string  | root directory used by Kpops to run pipelines                                                                 |
+| execute           |    ❌    |     false     | boolean | Execute Kpops command (this applies the infrastructure changes that were executed inside the dry-run command) |
```

Comment:
should it be `"false"`?

Reply:
Yes, it should be a conscious decision to deploy

Reply:
I meant if the type should be string

Reply:
here it is ok. It will be passed as string per default. You don't need to write `"false"` or `"true"` when you use the action.

Reply:
if it will be passed as string, please change the type descriptor

Reply:
like this?